algebra/src/main/java/org/almostrealism/collect/CollectionFeatures.java

```diff
@@ -2551,6 +2576,29 @@ default <T extends PackedCollection<?>> CollectionProducer<T> greaterThan(Produc
 		return (CollectionProducer<T>) new GreaterThanCollection(a, b, trueValue, falseValue, includeEqual);
 	}
 
+	/**
+	 * Performs element-wise equality comparison between two collections with custom return values.
+	 * This method compares corresponding elements and returns specified values based on the comparison result.
+	 * 
+	 * <p>This method is commonly used with {@link EpsilonConstantComputation} as the threshold parameter
+	 * for floating-point equality comparisons that need to account for numerical precision:</p>
+	 * <pre>{@code
+	 * // Compare two collections with epsilon tolerance
+	 * CollectionProducer<PackedCollection> areEqual = 
+	 *     equals(a, b, new EpsilonConstantComputation<>(shape), 
```

Comment:
This is not how the equals method of CollectionFeatures behaves. Just remove this discussion entirely; we can come back to it when documenting the comparison computations.